apps/calendar/js/views/pan.js

```diff
@@ -187,11 +192,12 @@ Pan.prototype = {
   },
 
   _set: function(x) {
+    var isRTL = (document.documentElement.dir === 'rtl') ? true : false;
```

Comment:
maybe we should just create a helper `utils/l10n.js`:

```js
define(function(require, exports) {
'use strict';

exports.init = function() {
  update();
  window.addEventListener('localized', update);
};

exports.isRTL = null;

function update() {
  exports.isRTL = document.documentElement.dir === 'rtl';
}

});
```

I can see more methods being added in the future and that way we can keep same API even if the way l10n is handled changes in the future.